functional_tests/features/methodology_page.feature

```diff
@@ -1,16 +1,16 @@
 Feature: A general use of Methodology Page
 
```

Comment:
We can utilise `Background` in this feature file and reduce the repetition of `Given a topic page exists under the homepage` in this feature file.

```suggestion
Background:
    Given a topic page exists under the homepage
```

Reply:
@sanjeevz3009 while tempting, I refrained from making changes outside of the scope of the ticket as it already changes quite a few files.

Reply:
All good. We have many more UI testing tickets where this can be cleaned up. Will chuck this comment in one of the UI tickets in the backlog. https://jira.ons.gov.uk/browse/CMS-329, added in the comments.

Resolving, thanks @zerolab